lib/ansible/modules/net_tools/netplan.py

```diff
@@ -0,0 +1,903 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2018, Luis Eduardo <leduardo@lsd.ufcg.edu.br>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: netplan
+author:
+    - Luis Eduardo (@lets00)
+short_description: Manage network configurations using netplan.
+version_added: "2.8"
+description:
+    - This module allows to you manage network configurations using netplan.
+      It creates/edits a netplan YAML file and saves on /etc/netplan/ directory
+      and apply on system.
+
+requirements: [ PyYAML, netplan ]
+notes:
+    - Online netplan page U(https://github.com/CanonicalLtd/netplan/blob/master/doc/netplan.md)
+options:
+  filename:
+    description:
+      - Name of file that will be create or open to add or remove network
```

Comment:
The description doesn't make sense.  Can you clarify it?

Reply:
If this is the path to the `netplan` file that will be created, I think the idiomatic parameter name to use would be `path`.